octavia_proxy/tests/functional/api/v2/base.py

```diff
@@ -145,3 +234,43 @@ def get(self, path, params=None, headers=None, status=200,
             expect_errors=expect_errors
         )
         return response
+
+    def post(self, path, body, headers=None, status=201, expect_errors=False,
+            use_v2_0=False, authorized=True):
+        headers = headers or {}
+        if use_v2_0:
+            full_path = self._get_full_path_v2_0(path)
+        else:
+            full_path = self._get_full_path(path)
+        if authorized:
+            if not headers:
+                headers = dict()
+            headers['X-Auth-Token'] = self._get_token()
+            body['loadbalancer']['project_id'] = self.project_id
+        response = self.app.post_json(full_path,
+                                    params=body,
+                                    headers=headers,
+                                    status=status,
+                                    expect_errors=expect_errors)
+        return response
+
+    def delete(self, path, headers=None, params=None, status=204,
+               expect_errors=False, authorized=True):
+        headers = headers or {}
+        params = params or {}
+        full_path = self._get_full_path(path)
+        param_string = ""
+        for k, v in params.items():
```

Comment:
this should be also not required, since we can use `params` or the request lib, but I might oversee something now